pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -234,6 +234,13 @@
             "#import": [ "rhel", "linux-x64" ]
         },
 
+        "rhel.6": {
```

Comment:
@ericstj can you please take a look at this RID change? I'm unsure if we want to update the rhel.7 to depend on rhel.6 now or not so I'd like a second opinion.